adminapp/src/pages/OfferingPickListPage.jsx

```diff
@@ -22,6 +22,7 @@ import map from "lodash/map";
 import sortBy from "lodash/sortBy";
 import uniqBy from "lodash/uniqBy";
 import React from "react";
+import { formatPhoneNumber } from "react-phone-number-input";
```

Comment:
Making a note we should probably kill this, and just return the formatted number from the backend.